.github/workflows/scancode-release.yml

```diff
@@ -131,22 +133,24 @@ jobs:
 
     strategy:
       fail-fast: true
+      matrix:
+       pyver: ["3.7", "3.8", "3.9", "3.10", "3.11"]
 
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up Python
+      - name: Set up Python on ${{ matrix.pyver }}
         uses: actions/setup-python@v1
         with:
-          python-version: "3.8"
+          python-version: ${{ matrix.pyver }} 
 
       - name: Build mac app archive
-        run: etc/release/scancode-create-release-app-macos.sh
+        run: etc/release/scancode-create-release-app-macos.sh ${{ matrix.pyver }}
 
-      - name: Collect built mac app
+      - name: Collect built mac app for python {{ matrix.pyver }}
```

Comment:
See https://github.com/AyanSinhaMahapatra/scancode-toolkit/actions/runs/4046908202/jobs/6960286886#step:5:1, this is being rendered as `Collect built mac app for python {{ matrix.pyver }}` but see https://github.com/AyanSinhaMahapatra/scancode-toolkit/actions/runs/4046908202/jobs/6960286886#step:3:1 where it is being rendered correctly in the `name`

Reply:
my bad I missed a $, this is fixed now